docs/examples/FinetuneOlmoEarth.md

```diff
@@ -0,0 +1,279 @@
+rslearn dataset add_windows --root ./dataset --group default --utm --resolution 10 --src_crs EPSG:4326 --box=-122.0,47.8,-121.9,47.9 --start 2024-05-01T00:00:00+00:00 --end 2024-09-01T00:00:00+00:00 --grid_size 128
+rslearn dataset add_windows --root ./dataset --group default --utm --resolution 10 --src_crs EPSG:4326 --box=-122.1,47.6,-122.0,47.7 --start 2024-05-01T00:00:00+00:00 --end 2024-09-01T00:00:00+00:00 --grid_size 128
+rslearn dataset add_windows --root ./dataset --group default --utm --resolution 10 --src_crs EPSG:4326 --box=-122.8,47.7,-122.7,47.8 --start 2024-05-01T00:00:00+00:00 --end 2024-09-01T00:00:00+00:00 --grid_size 128
+rslearn dataset add_windows --root ./dataset --group default --utm --resolution 10 --src_crs EPSG:4326 --box=-122.6,47.6,-122.5,47.7 --start 2024-05-01T00:00:00+00:00 --end 2024-09-01T00:00:00+00:00 --grid_size 128
+rslearn dataset add_windows --root ./dataset --group default --utm --resolution 10 --src_crs EPSG:4326 --box=-123.2,47.2,-123.1,47.3 --start 2024-05-01T00:00:00+00:00 --end 2024-09-01T00:00:00+00:00 --grid_size 128
+rslearn dataset add_windows --root ./dataset --group default --utm --resolution 10 --src_crs EPSG:4326 --box=-122.1,47.2,-122.0,47.3 --start 2024-05-01T00:00:00+00:00 --end 2024-09-01T00:00:00+00:00 --grid_size 128
+```
+
+Use rslearn to materialize the data:
+
+```
+rslearn dataset prepare --root ./dataset --workers 32 --retry-max-attempts 5 --retry-backoff-seconds 5
+rslearn dataset ingest --root ./dataset --workers 32
+rslearn dataset materialize --root ./dataset --workers 32 --retry-max-attempts 5 --retry-backoff-seconds 5 --ignore-errors
+```
+
+Split the data into train and val:
+
+```python
+import hashlib
+import tqdm
+from rslearn.dataset import Dataset, Window
+from upath import UPath
+
+ds_path = UPath("./dataset")
+dataset = Dataset(ds_path)
+windows = dataset.load_windows(show_progress=True, workers=32)
+for window in tqdm.tqdm(windows):
+    if hashlib.sha256(window.name.encode()).hexdigest()[0] in ["0", "1"]:
+        split = "val"
+    else:
+        split = "train"
+    if "split" in window.options and window.options["split"] == split:
+        continue
+    window.options["split"] = split
+    window.save()
+```
+
+## Fine-tune the Model
+
+We will use this model configuration file for fine-tuning OlmoEarth. See the comments
+below for details on how each part functions. [The OlmoEarth rslearn reference](../foundation_models/OlmoEarth.md)
+also has more information.
+
+```yaml
+model:
+  class_path: rslearn.train.lightning_module.RslearnLightningModule
+  init_args:
+    model:
+      class_path: rslearn.models.singletask.SingleTaskModel
+      init_args:
+        encoder:
+          # This applies the OlmoEarth encoder on the inputs.
+          - class_path: rslearn.models.olmoearth_pretrain.model.OlmoEarth
+            init_args:
+              model_id: OLMOEARTH_V1_BASE
+              patch_size: 8
```

Comment:
nit: it might be nice to copy the comments over from OlmoEarth.md for those who miss the link above